src/modules/aws-ebs/sub-modules/config/loadEbsConfig.ts

```diff
@@ -0,0 +1,42 @@
+import { EbsConfig } from "./EbsConfig";
+
+export const isEnvironmentVariableDefined = (environmentVariable: string | undefined): boolean => {
+  return (environmentVariable && environmentVariable !== "undefined") === true;
+};
+
+export const loadEbsConfig = (): EbsConfig => {
+  let sourceAwsRegion: string;
+
+  if (process.env.AWS_REGION && process.env.AWS_REGION !== "undefined") {
+    sourceAwsRegion = process.env.AWS_REGION;
+  } else {
+    throw new Error("Env variable AWS_REGION must be defined.");
+  }
+
+  const quarantineAwsRegion: string =
+    process.env.QUARANTINE_AWS_REGION && process.env.QUARANTINE_AWS_REGION !== "undefined"
+      ? process.env.QUARANTINE_AWS_REGION
+      : process.env.AWS_REGION;
+
+  const quarantineAwsAccounts: string[] =
+    process.env.QUARANTINE_AWS_ACCOUNTS && process.env.QUARANTINE_AWS_ACCOUNTS !== "undefined"
+      ? process.env.QUARANTINE_AWS_ACCOUNTS.split(",").map(account => {
+          return account.trim();
+        })
+      : [];
+
+  const transferAllSnapshots: boolean =
+    process.env.TRANSFER_ALL_SNAPSHOTS && process.env.TRANSFER_ALL_SNAPSHOTS !== "undefined"
+      ? process.env.TRANSFER_ALL_SNAPSHOTS == "true"
+      : false;
+
+  if (!process.env.QUARANTINE_AWS_ACCOUNTS || process.env.QUARANTINE_AWS_ACCOUNTS === "undefined") {
```

Comment:
Should this be cleaned up?

Reply:
ah yeah. env variables are a little weird. typescript says this can either be string or undefined. however, if an env variable wasn't defined, calling it returns a string representation for "undefined". that's why i needed to do an implicit check `==="undefined"`.

Reply:
lol